### PR TITLE
Gradle 5.4

### PR DESCRIPTION
**Pull request by skydrome**

### CODE REVIEW

**Comment by skydrome** [comment]
https://travis-ci.org/skydrome/fred/builds/521945386 example of travis options uncommented
https://ci.appveyor.com/project/skydrome/fred example of appveyor windows ci
https://sonarcloud.io/organizations/skydrome-github/projects example of sonarcloud

**Comment by skydrome** [comment]
a suggestion:
maintain a fork of the gradle-witness plugin under the freenet repos

**Comment by ArneBab** [comment]
I’ll still be overloaded for a bit more than two weeks, afterwards I hope I’ll be able to review better. Thank you!

File: .gitignore [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+.idea/
 *.iml
 *.ipr
 *.iws
-.*
-*~
-\#*\#
-/override.properties
-/gradle.properties
+*.swp
```
Comment (Bombe) [inline .gitignore]: While I totally agree that the repository should not contain any files that are not relevant, I am of the opinion that we only need to exclude files in `.gitignore` that are actually created by the build system. It’s every developer’s own responsibility to not check in their editor’s, IDE’s, or OS’s temp or backup files that might end up in the working copy of the repository. (Git can use a global `.gitignore` file, too.)

**Pull request closed without merging**